apps/app/src/app/tasks/feature/tasks-list/tasks-list.page.ts

```diff
@@ -197,6 +197,35 @@ export class TasksListComponent implements OnInit {
     return this._settingsService.initialInterval;
   }
 
+  public getStatusLabel(status: number): string {
```

Comment:
Je ne peux pas référencer le problème sur la bonne ligne, je le fais donc ici.

Il faudrait utiliser cette fonction ligne `175` afin que le `select filter` puisse avoir des labels traduisibles.
Il faudrait modifier la ligne ainsi:
`label: this.getStatusLabel(TaskStatus[key as keyof typeof TaskStatus]),`